0009/README.md

```diff
@@ -0,0 +1,70 @@
+---
+id: 0009
+title: Automated GitHub Action Runner
```

Comment:
Can you change this? I think we should stay a bit more general about the test-infra repo itself. The GitHub action runner is just one (probably the first) way we will use it